src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((block) => block.trim())
+    .filter((block) => block.includes(':'))
+    .reduce((cssProperties, block) => {
+      const [key, value] = block
+        .split(':')
+        .map((splitBlock) => splitBlock.trim());
+
+      cssProperties[key] = value;
+
+      return cssProperties;
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Ensure that the methods `split`, `map`, `filter`, and `reduce` are all aligned at the same indentation level.